index.html

```diff
@@ -0,0 +1,240 @@
+            <li><p>Back End Dev</p></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+      <li class="item tonic">
+        <img src="../Portfolio-finish-mobile-version/img/work3.svg" alt="">
+        <div>
+          <div>
+          <h1>Tonic</h1>
+          <ul class="canopy">
+            <li><h3>CANOPY</h3></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>Back End Dev</p></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+      <li class="item multi-post-stories">
+        <img src="../Portfolio-finish-mobile-version/img/work4.svg" alt="">
+        <div>
+          <div>
+          <h1>Multi-Post Stories</h1>    
+          <ul class="canopy">
+            <li><h3>CANOPY</h3></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>Back End Dev</p></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+    </ul>
+    </main>
```

Comment:
- Hey, to enforce good use of html5 semantic tag, kindly move this `main` tag that is nested in the section tag out of it. As the `main` tag you've used is supposed to be a `section` tag which denotes another section of your website so it should be a standalone part of your website. kindly rename the main to section to meet the project requirement.